cg/cli/workflow/microsalt/base.py

```diff
@@ -122,6 +122,8 @@ def config_case(context: click.Context, dry_run: bool, ticket: int, sample_id: s
 def run(context: click.Context, dry_run: bool, config_case_path: click.Path, ticket: int):
     """ Start microSALT with an order_id """
 
+    microbial_api = context.obj["analysis_api"]
```

Comment:
Could you re-name this object to be more microsalt-specific? Such as microsalt_analysis_api, to keep it more distinct form other analysis api we have in cg